rita_common/src/payment_controller/mod.rs

```diff
@@ -143,10 +156,176 @@ pub async fn tick_payment_controller() {
 
 /// This is called by debt_keeper to make payments. It sends a
 /// PaymentTx to the `mesh_ip` in its `to` field.
-async fn make_payment(mut pmt: PaymentTx) -> Result<(), PaymentControllerError> {
+async fn make_payment(pmt: PaymentTx) -> Result<(), PaymentControllerError> {
     let common = settings::get_rita_common();
     let network_settings = common.network;
     let payment_settings = common.payment;
+    let system_chain = payment_settings.system_chain;
+
+    match system_chain {
+        SystemChain::Althea => make_althea_payment(pmt, payment_settings, network_settings).await,
+        SystemChain::Xdai => make_xdai_payment(pmt, payment_settings, network_settings).await,
+        SystemChain::Rinkeby => {
+            warn!("Payments on Rinkeby not currently supported!");
```

Comment:
Rinkeby and Ethereum can work with the same flow as before I see no reason to break them.

Reply:
There were no flows before for rinkeby and eth, this function previously only did xdai payments
I added the match here to determine to do an xdai flow or althea flow now since we support two different chains